### DISCUSSION TITLE
Subtitles start at 00:00:00 but voice starts later

### DISCUSSION THREAD

**Discussion by alhafoudh** [openai/whisper#601] [Q&A]
I try to transcribe podcast audio. Many audio samples that I have start with some music and after few seconds the human voice starts. In this case first subtitle segment starts at 00:00:00 but should start few seconds later. Any idea how to tweak this so the first subtitle segment starts with the voice?

**Comment by orionflame** [comment]
I have the same issue but not only that, for all text where is a gap, the text appears way before even when there is no audio. So the start timings are not correct.

I am using the default write_srt function so it would be nice if this was improved.

**Comment by ThioJoe** [comment]
The issue is that it does not generate gaps between any of the timings. Every start and end point for the different lines but right up against each other.

**Comment by 0xDebabrata** [comment]
Take a look at #435 
`stable-ts` provides a great improvement over the default Whisper timestamps.
